.github/workflows/docker-image.yml

```diff
@@ -1453,6 +1453,69 @@ jobs:
         if: always()
         run: sudo podman logs fetchit
 
+  ssh-validate:
+    runs-on: ubuntu-latest
+    needs: [ build, pull-and-archive ]
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: pull in podman
+        uses: actions/download-artifact@v1
+        with:
+          name: podman-bins
+          path: bin
+
+      - name: replace
+        run: |
+          chmod +x bin/podman
+          sudo mv bin/podman /usr/bin/podman
+
+      - name: Enable the podman socket
+        run: sudo systemctl enable --now podman.socket
+
+      - name: pull artifact
+        uses: actions/download-artifact@v1
+        with:
+          name: fetchit-image
+          path: /tmp
+
+      - name: pull artifact
+        uses: actions/download-artifact@v1
+        with:
+          name: colors
+          path: /tmp
+
+      - name: Load the image
+        run: sudo podman load -i /tmp/fetchit.tar
+
+      - name: Load the image
+        run: sudo podman load -i /tmp/colors.tar
+
+      - name: tag the image
+        run: sudo podman tag quay.io/fetchit/fetchit-amd:latest quay.io/fetchit/fetchit:latest
+
+      - name: generate ssh assets
+        run: |
+          mkdir -p ~/.fetchit/.ssh
+          cp ./examples/ssh-config.yaml ~/.fetchit/config.yaml
+          echo ${{ secrets.SSH_PRIVATE_KEY }} > ~/.fetchit/.ssh/id_rsa
+          ssh-keyscan github.com > ~/.fetchit/.ssh/known_hosts
+          chmod 0600 -R ~/.fetchit/.ssh 
+
+      - name: Start fetchit
+        run: sudo podman run -d --name fetchit -v fetchit-volume:/opt -v /home/runner/.fetchit:/opt/mount/config.yaml -v /run/podman/podman.sock:/run/podman/podman.sock --security-opt label=disable quay.io/fetchit/fetchit-amd:latest
```

Comment:
@djach7 
add this step:
```
      - name: generate fetchit config directory
        run: |
            mkdir -p /home/runner/.fetchit
            cp examples/ssh-config.yaml /home/runner/.fetchit/config.yaml
```
and in the podman run cmd,         
update `-v /home/runner/.fetchit:/opt/mount/config.yaml` to `-v /home/runner/.fetchit:/opt/mount`

Reply:
It looks like the last run failed with some permissions issues. May just need to dump sudo in front of the commands then do a chown afterwards as well